lib/diagnostic.rb

```diff
@@ -100,13 +100,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.delete(2,3)'
 
 # What would `arr` equal after:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = '[12, 34, 56, 67,,99]'
```

Comment:
You'll have two `nil` in there since it's `.length + 2`

`Diagnostic.array_add_to = '[12, 34, 56, 67, nil, nil, 99]'`